doc/build-netbsd.md

```diff
@@ -35,16 +35,16 @@ BerkeleyDB is only necessary for the wallet functionality. To skip this, pass
 It is recommended to use Berkeley DB 4.8. You cannot use the BerkeleyDB library
```

Comment:
Looks like this line needs an update too.

Do we know if the bdb library in ports still has this limitation? (That's question shouldn't block this PR.)